source/testing/testing-helpers.md

```diff
@@ -36,59 +36,65 @@ are functions, which can be easily tested with `module`.
 import { formatCurrency } from 'my-app/helpers/format-currency';
 import { module, test } from 'qunit';
 
-module('Unit | Helper | format currency');
+module('Unit | Helper | format currency', function(hooks) {
+  setupTest(hooks);
```

Comment:
if you explicitly import the helper then you don't need `setupTest()`. looks like the official blueprint is currently wrong too about this (https://github.com/emberjs/ember.js/blob/v3.0.0-beta.6/blueprints/helper-test/qunit-rfc-232-files/tests/__testType__/helpers/__name__-test.js#L17-L22)

/cc @snewcomer @rwjblue

Reply:
Thanks for pointing that one out, I literally just copied that over from the blueprint as well 👍